roofit/hs3/CMakeLists.txt

```
@@ -0,0 +1,48 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+############################################################################
+# CMakeLists.txt file for building ROOT rootfit/histfactory package
+# @author Carsten Burgard, CERN
+############################################################################
+
+ROOT_STANDARD_LIBRARY_PACKAGE(HS3
```

```suggestion
ROOT_STANDARD_LIBRARY_PACKAGE(RooFitHS3
```
Please prefix the package name with `RooFit` so it's clear what this is (analogous to `RooFitCore` package).